lenses_python/SqlHandler.py

```diff
@@ -100,7 +103,7 @@ def browsing_data(self, stats=0,  is_extract_pandas=False):
         if not is_extract_pandas:
             return {"records": data_list,
                     "stats": stats_list
-                   }
+                    }
```

Comment:
you have extra space. there is no need tor two line in general here. except if you exceed line limit by pep8